Apps/WebClient/src/ClientApp/src/store/modules/auth/getters.ts

```diff
@@ -60,4 +60,9 @@ export const getters: GetterTree<AuthState, RootState> = {
     oidcError(state: AuthState): any {
         return state.error;
     },
+    isBcscLogin(state: AuthState): boolean {
```

Comment:
What is the purpose of this check?  I think we want to specifically determine if IDIR not BCSC as we don't want to prevent Keycloak logins and we don't want to prevent future IDPs.

Reply:
At this point the only valid IDP is bcsc, so that is what this is checking. If we get another provider we could change it in the future.

Reply:
updated to check for IDIR